draft-ietf-jsonpath-base.md

```diff
@@ -296,7 +290,8 @@ stable JSON query language.
 
 ## Overview of JSONPath Expressions {#overview}
 
-JSONPath expressions are applied to a JSON value, the *argument*.
+A JSONPath expression is applied to a JSON value, the *argument*, and
```

Comment:
I'd like a second sentence instead of an /and/, but this is just preference,

Reply:
Yes, I like that. I'll submit a PR.